tequilapi/endpoints/stop.go

```diff
@@ -0,0 +1,20 @@
+package endpoints
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"net/http"
+)
+
+// Stopper stops application and performs required cleanup tasks
+type Stopper func()
+
+// AddRouteForStop adds stop route to given router
+func AddRouteForStop(router *httprouter.Router, stop Stopper) {
```

Comment:
We are in very deep layer, what king of stopper is it?
Too generic name is it program stopper?

Reply:
king of stopper?

Reply:
maybe `ApplicationStopper`

Reply:
done